.gitignore

```diff
@@ -1,3 +1,2 @@
 .idea
-composer.lock
```

Comment:
Agreed; there is no policy currently but as we are typing I think we should set up a developer page and a few crons that check this for all repositories in the phpDocumentor organisation